runchecks

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+show_help() {
+    printf "Usage:\n"
+    printf "runchecks [--help] [--skip-checkendline] [--skip-checkmigration] [--skip-flake8] [--skip-isort] [--skip-checkcommit]\n"
+    printf "\n\n"
+    printf "Options:\n"
+    printf "  --help\tshow this help text\n"
+    printf "  --skip-checkendline\tskip endline check\n"
+    printf "  --skip-checkmigration\tskip migration check\n"
+    printf "  --skip-flake8\tskip flake8\n"
+    printf "  --skip-isort\tskip isort\n"
+    printf "  --skip-checkcommit\tskip commit check\n"
+}
+
+runcheckendline() {
+    flag=0
+    for i in $(find . -type f ! -path "*/*.egg-info/*" \
+        ! -path "./.*" \
+        ! -path "*.min.*" \
+        ! -path "*.svg" -exec grep -Iq . {} \; -and -print); do
+        if [ "$(tail -c 1 $i)" != "" ]; then
+            echo "$i needs newline at the end"
+            flag=1
+        fi
+    done
+    if [ $flag -ne 0 ]; then
+        exit $flag
+    fi
+}
+
+runcheckmigration() {
+    set -e
+    checkmigrations --migration-path ./tests/test_project/migrations/ || exit 1
```

Comment:
The arguments aren't being passed correctly. Can you take a look?
https://travis-ci.org/openwisp/openwisp-utils/jobs/621280059#L427

Reply:
The thing with passing arguments turned up to be not the problem with my script. It's a problem with coverage. You can see some [old build](https://travis-ci.org/openwisp/openwisp-utils/jobs/525184823#L636) (even without `runchecks`) where these errors are still appearing. Presumably, `runchecks.py` runs `checkcommit` and `checkmigration` functions, that have no arguments to process which results in an error. I can take a deeper look into it if you wish.